meinberlin/apps/cms/templates/meinberlin_cms/emails/form_submission.en.email

```diff
@@ -7,4 +7,12 @@
 
 {% block content %}
     {{ emailformpage.email_content }}
+
+    {% for key, value in field_values.items %}
+
+        {{key}}:
+
```

Comment:
i would skip this empty line. looks weird and makes the field (keyvalue) blocks less distinguishable